### ISSUE TITLE
Instructions for installing the resource pack generator via pipx

### ISSUE THREAD

**Issue by OpenBagTwo** [OpenBagTwo/FoxNap#64]
Blocked by #63 

It's always been problematic that `FoxNapRPG` is distributed as an EXE (see: #62). Those who have some comfort with the command-line should have the ability to install it via `pip` (explicitly: [`pipx`](https://pipx.pypa.io/stable/)).

## Notes
- Documentation should go in the wiki (but linked to from the README)
- Builds published to PyPI is a nice-to-have but not required
- Instructions must include system-wide installation of ffmpeg-full
- Instructions must be tested on Windows

**Comment by OpenBagTwo** [comment]
Instructions now in the wiki
